react-native/react-native.d.ts

```diff
@@ -1483,6 +1483,14 @@ declare namespace  __React {
          */
         injectedJavaScript?: string
 
+        onError?:  ( event: NavState ) => void
```

Comment:
Can you add comments? Something like
/*
Invoked when load fails
*/

See https://facebook.github.io/react-native/docs/webview.html